tests/integration/test_vulnerability_detector/test_feeds/test_download_feeds.py

```diff
@@ -31,6 +31,7 @@
     - Arch Linux
     - Amazon Linux 2
     - Amazon Linux 1
+    - Amazon Linux 2022
```

Comment:
```suggestion
    - Amazon Linux 2022
    - Amazon Linux 2
    - Amazon Linux 1
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/2955/commits/47bba0cb4c062403163cf3acfe3ea5a9f2220ddb